Source/WebCore/Modules/applepay/PaymentSessionError.h

```diff
@@ -44,6 +45,7 @@ class WEBCORE_EXPORT PaymentSessionError {
     NSError *platformError() const;
 
 private:
+    friend struct IPC::ArgumentCoder<PaymentSessionError, void>;
```

Comment:
I think it would be better instead to make platformError return a RetainPtr<NSError>